openedx/core/djangoapps/user_api/accounts/serializers.py

```diff
@@ -63,6 +63,7 @@ def to_representation(self, user):
         :return: Dict serialized account
         """
         profile = user.profile
+        has_accomplishments = settings.FEATURES.get('ENABLE_OPENBADGES') or False
```

Comment:
Shouldn't `has_accomplishments` actually return whether or not the user has acquired badges?